.editorconfig

```diff
@@ -0,0 +1,14 @@
+# Comma-separated list of rules to disable (Since 0.34.0)
+# Note that rules in any ruleset other than the standard ruleset will need to be prefixed
+# by the ruleset identifier.
+disabled_rules=import-ordering,no-wildcard-imports,unused-import
```

Comment:
Unused import seems useful. At least as a warning.

Import ordering would also probably be nice, going forward. But then it might be weird having all new files adhere to the ordering while old files do not.

Perhaps we can get this PR merged and then decide on an import ordering and then apply that to all files as a separate PR?